2dcontext/tools/tests2d.yaml

```diff
@@ -1007,41 +1007,149 @@
 
 
 - meta: |
-    # Composite operation tests
-    # <http://lists.whatwg.org/htdig.cgi/whatwg-whatwg.org/2007-March/010608.html>
     ops = [
-        # name               FA      FB
-        ('source-over',      '1',    '1-aA'),
-        ('destination-over', '1-aB', '1'),
-        ('source-in',        'aB',   '0'),
-        ('destination-in',   '0',    'aA'),
-        ('source-out',       '1-aB', '0'),
-        ('destination-out',  '0',    '1-aA'),
-        ('source-atop',      'aB',   '1-aA'),
-        ('destination-atop', '1-aB', 'aA'),
-        ('xor',              '1-aB', '1-aA'),
-        ('copy',             '1',    '0'),
-        ('lighter',          '1',    '1'),
+        # Composite operation tests
+        # <http://lists.whatwg.org/htdig.cgi/whatwg-whatwg.org/2007-March/010608.html>
```

Comment:
This reference is out of date. The current version of the whatwg spec references the CSS blending and Compositing spec for enumerating available modes.
There are trivial compositing modes in that spec that are missing here: destination and clear. Also, there are the plus-darker and plus-lighter modes